third_party/patches/bazel_gazelle/label-pattern-matching.patch

```diff
@@ -0,0 +1,388 @@
+commit dfed19229cbcec18d2338d92561b2ef2cd6f7384
+Author: Daniel Wagner-Hall <dwagnerhall@apple.com>
+Date:   Tue Apr 5 14:57:09 2022 +0100
+
+    Add label pattern matching logic
+    
+    This implements target patterns as described at https://bazel.build/docs/build#specifying-build-targets
+    
+    It is not immediately used in Gazelle, but as Gazelle has a solid label
```

Comment:
same, is there an upstream PR so this patch has a chance of going away?

Reply:
Yeah, this one is https://github.com/bazelbuild/bazel-gazelle/pull/1227

Reply:
could you include that in the filename so it's easier to trace?